File: Javascript/stack.js
function Stack() {
    let items = [];

    this.push = function (element) {
        items.push(element);
    }
    this.pop = function () {
        items.pop()
    }
    this.peek = function () {
        return items[items.length - 1];
    }
    this.size = function () {
        return items.length;
    }
    this.clear = function () {
        items = [];
    }
    this.isEmpty = function () {
        return items.length === 0;
    }
    this.print = function () {
        console.log(items.toString());
    }
}

let stack = new Stack();
stack.push(1)
stack.push(3)
stack.push(9)
stack.print()
stack.push(5)
stack.push(22)
stack.print()
stack.pop()
stack.print()
stack.isEmpty()
console.log(stack.peek())
console.log(stack.size())
stack.pop()
stack.print()
stack.isEmpty()
console.log(stack.peek())
console.log(stack.size())

File: Python/stack.py
import json

class Stack():
    def __init__(self):
        self.items = []

    def isEmpty(self):
        return self.items == []

    def push(self, item):
        self.items.append(item)
    
    def pop(self):
        if not self.isEmpty():
            return self.items.pop()
        else:
            raise Exception('stack is empty!')

    def peek(self):
        if not self.isEmpty():
            return self.items[-1]
        else:
            raise Exception('stack is empty!')
        
    def size(self):
        if not self.isEmpty():
            return len(self.items)
        else:
            raise Exception('stack is empty!')   

    def toJson(self):
        return json.dumps(self.items)

def main():
    stack = Stack()
    stack.push(22)
    stack.push(44)
    stack.push(93)
    print(stack.size())
    print(stack.toJson())
    print(stack.pop())
    print(stack.size())
    print(stack.toJson())
    print(stack.pop())
    print(stack.toJson())
    print(stack.size())
    print(stack.pop())
    print(stack.toJson())

if __name__ == "__main__":
    main()